### PR TITLE
Start work to prepare next 2.0.x patch

### PR DESCRIPTION
**Pull request by natemcmaster**
Updates the 2.0.x build to include patch changes to Kestrel and EF Core.

cc @Eilon

### CODE REVIEW

**Comment by natemcmaster** [comment]
cc @jkotalik

File: version.props [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.0.4</VersionPrefix>
+    <VersionPrefix>2.0.5</VersionPrefix>
```
Comment (Eilon) [inline version.props]: Oh wait, wasn't the release we just did 2.0.5? And Feb is 2.0.6?

Comment (natemcmaster) [reply version.props]: Yup. I'm waiting to merge this until we get those changes into the public branch. Then I'll bump to 2.0.6

Comment (Eilon) [reply version.props]: Ah ok thanks for clarifying.

**Comment by bricelam** [comment]
Can you update Microsoft.Data.Sqlite as part of this? (to react to aspnet/Microsoft.Data.Sqlite#475)

**Comment by natemcmaster** [comment]
@bricelam will do

File: build/dependencies.props [hunk 2]
```diff
@@ -391,7 +392,7 @@ not building again in this patch.
     <ExternalDependency Include="Microsoft.AspNetCore.Rewrite" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
     <ExternalDependency Include="Microsoft.AspNetCore.Routing.Abstractions" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
     <ExternalDependency Include="Microsoft.AspNetCore.Routing" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
-    <ExternalDependency Include="Microsoft.AspNetCore.Server.HttpSys" Version="2.0.1" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
+    <ExternalDependency Include="Microsoft.AspNetCore.Server.HttpSys" Version="2.0.2" Source="$(DefaultNuGetFeed)" Metapackage="true" RuntimeStore="true"/>
```
Comment (muratg) [inline build/dependencies.props]: Not sure why this change is just now happening -- it's 2.0.2 on nuget for a while.

**Comment by natemcmaster** [comment]
Rebase this on 2.0.5 changes now

File: build/dependencies.props (hunk 2 above)
Comment (natemcmaster) [reply build/dependencies.props]: 2.0.2 was released out-of-band from the metapackage. We didn't have proper follow-thru to update the metapackage in the subsequent release. I just noticed now as I was scanning for latest package versions.
